### DISCUSSION TITLE
question about output

### DISCUSSION THREAD

**Discussion by AmosJoseph** [lululxvi/deepxde#868] [General]
[Multi-fidelity learning from the formulas](https://github.com/lululxvi/deepxde/blob/master/examples/function/mf_func.py)

Pardon, what's the meaning of output below:
step     Train loss                                       Test loss                                          Test metric
0         [4.02e+01, 5.22e+01, 1.95e+00]    [8.21e+01, 2.41e+01, 1.95e+00]    [1.00e+00, 1.04e+00]    

There are 3 items for Train loss and Test loss. Why? 3 items are  about high, low, and multi fidelity?

Test metric has 2 items? What are them? 
The mean and standard deviation for the relative L2 errors?

sorry to bother！

**Comment by lululxvi** [comment]
- losses: low-fidelity loss, high-fidelity loss, regularization
- Test metric: l2 relative error for low- and high-fidelity prediction
